packages/foam-cli/src/utils/rename-file.ts

```diff
@@ -0,0 +1,11 @@
+import * as fs from 'fs'
+
+
+
+export const renameFile = async (fileUri: string, newFileName: string) => {
+  const fileName = fileUri.split('/').pop();
```

Comment:
This may not work on Windows -- would depend whether URIs are normalised before passing. There's probably a node x-plat method for doing this

Reply:
Done!